### ISSUE TITLE
Unable to download the Panda3d

### ISSUE THREAD

**Issue by toonninja66** [eccentricelephanteatingeasels/ttoff-src#1]

**Comment by eccentricelephanteatingeasels** [comment]
yes looks like the downloads are dead. panda3d for windows can be found here https://mega.nz/#!SxNE3QQa!60nOBBD342fBhJ69rpDlcS9hc19ay2g-Vy3DiSjyFuM

"life is too short to be an asshole"
カンナカムイ

**Comment by toonninja66** [comment]
Alright thanks. I appreciate it. What's your discord? I feel like we could be good friends.

**Comment by toonninja66** [comment]
![image](https://user-images.githubusercontent.com/16170893/56701962-a12f3600-66cf-11e9-8f28-ef1abb00868c.png)
I also have this issue whenever I create a toon

**Comment by eccentricelephanteatingeasels** [comment]
hi sorry for inconvenience. just got same error myself. i fixed the issue and you will need to download new panda3d which is here https://mega.nz/#!bod3kYqT!VRzwZqorzl5VTctMoFyc0E635v7G6aLfoeqOgr0iBKY

also sorry do not have discord.

"the world owes you nothing"
フェリックス・アーガイル

**Comment by toonninja66** [comment]
Alright thanks. Does it also fix duck hunt issues?

**Comment by eccentricelephanteatingeasels** [comment]
"duck hunt" is very broken but not fault of panda3d.

**Comment by toonninja66** [comment]
One thing I've noticed from the source too is that the episodes don't work.

**Comment by eccentricelephanteatingeasels** [comment]
remember this is development repository so bugs will exist. repository is provided as is and i cannot fix any bugs.

**Comment by toonninja66** [comment]
Yeah. Thanks for the notes. Honestly I only assumed they would work though because they're in the actual TTOFF
